dev/Generated/NumberBox.properties.h

```diff
@@ -82,6 +85,7 @@ class NumberBoxProperties
     static winrt::DependencyProperty SmallChangeProperty() { return s_SmallChangeProperty; }
     static winrt::DependencyProperty SpinButtonPlacementModeProperty() { return s_SpinButtonPlacementModeProperty; }
     static winrt::DependencyProperty TextProperty() { return s_TextProperty; }
+    static winrt::DependencyProperty TextAlignmentProperty() { return s_TextAlignmentProperty; }
```

Comment:
I think this would technically go through API review first. @MikeHillberg Should this be marked as preview until then?